### DISCUSSION TITLE
sofaunity in asset store doesnt work like the github release version

### DISCUSSION THREAD

**Discussion by mariamgamal70** [InfinyTech3D/SofaUnity#134] [Help - Setting up SofaUnity]
so i downloaded sofa unity through unity asset store and it had everything but the interactions and tearing and burning effects didn't work and there were several errors in gamemode. so I downloaded the github release version and it works fine and very little errors in gamemode which is good but seems something is wrong with materials or lightening
the following is the asset store version:
![image](https://github.com/user-attachments/assets/ef9eb8e2-4501-4f6d-a2fc-6668cf2f17c2)
the following is the github version:
![image](https://github.com/user-attachments/assets/3b2f1e9c-94a0-4287-936c-c0cbd42ea265)
any reason why that could be ?

**Comment by epernod** [comment]
Hi,

thank you for the report. Could you detail which version of unity you are using with which rendering pipeline?
Also which package of libraries did you download to finish installation from the asset store version?
Then which release did you download on Github ?

Except from the lightning issue which might be related to your global environment lightning settings, did you experience others errors?

Thank you

**Reply by mariamgamal70** [reply]
As i mentioned the interactions weren't working in the examples or demo scenes unlike the github version. Thats what i discovered till now...

**Reply by mariamgamal70** [reply]
As for unity its unity 6, the github bersion that worked was v24.06 the latest one
I just downloaded it from asset store and imported it into my project on unity , there was no instructions that i should install anything after that..
Im currently using the github version just fine for now ..

**Reply by epernod** [reply]
Hi,

sorry for the late answer. We are not using Unity6 for the moment. But apart from material or prefabs that might not be compatible, the simulation engine should not be affected.

A new version v24.12 will be released in a few days. I suggest to use the github version it has less restriction than the version published on the Unity store.

Best regards,
